src/workspaces/ShareWorkspaceModal/ShareWorkspaceModal.tsx

```diff
@@ -92,20 +101,24 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
   )(groups);
 
   const remainingSuggestions = _.difference(suggestions, _.map('email', acl));
+  const addUserReminder =
+    'Did you mean to add collaborators? Add them or clear the "User email(s)" field to save changes.';
```

Comment:
Nit: It is just "User emails" now, without the parentheses.

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5205/commits/891eebc49784af506850523357fd9df3e7165bcb